### DISCUSSION TITLE
What is the thinking behind removing formatting of image alt texts?

### DISCUSSION THREAD

**Discussion by rauschma** [jgm/djot#298] [Q&A] [answered]
Since it’s clear that the image alt text is plain text, it could simply be parsed as plain text. I wonder why that isn’t done. (Not that it makes much of a difference. I’m simply curious.)

Input:

```md
![_abc_]()
```

Output (no underscores):

```html
<p><img alt="abc" src=""></p>
```

**Comment by jgm** [comment, marked as answer]
HTML can't have formatting in the alt text, because it's in an attribute. But that's a parochial feature of HTML output. If we think of this feature as an 'image description', there's no reason to prevent it from having formatting, which might be used in other output formats.  The formatting is stripped out in the HTML renderer.
